vm-validator/src/unit_tests/vm_validator_test.rs

```diff
@@ -166,15 +167,14 @@ fn test_validate_max_gas_units_above_max() {
 
 #[test]
 fn test_validate_max_gas_units_below_min() {
-    let (config, key) = config_builder::test_config();
-    let vm_validator = TestValidator::new(&config);
+    let vm_validator = TestValidator::new();
 
     let address = account_config::libra_root_address();
     let transaction = transaction_test_helpers::get_test_signed_transaction(
         address,
         1,
-        &key,
-        key.public_key(),
+        &vm_genesis::GENESIS_KEYPAIR.0,
```

Comment:
I guess any key pair works here? using genesis keypair looks weird

Reply:
for this one, I guess it needs to be a legitimate account? i dunno, but look at the address, so we get the key for that address... I don't really understand how the test works.